docs/en/Running-Alluxio-on-a-Cluster.md

```diff
@@ -200,6 +200,31 @@ Alluxio leader master.
 hadoop fs -ls alluxio:///directory
 ```
 
+##### Alluxio on Zookeeper URI
```

Comment:
Can we combine this with the previous two sections and discuss (1) how users should decide between using conf-based vs uri-based HA config (2) how to do it (3) details about what happens when both are configured.

Also, there shouldn't be anything specific to HDFS - anything that works for HDFS should also work the same way for native API, e.g. shell commands